packages/jest-cli/src/watch.js

```diff
@@ -195,8 +201,14 @@ const watch = (
         );
         break;
       case KEYS.QUESTION_MARK:
-        if (process.env.JEST_HIDE_USAGE) {
+        break;
```

Comment:
It appears that the intention behind `w` is the same as `?`. IMO either `?` should be removed, or they should be treated identically.